packages/components/src/tabs/test/index.tsx

```diff
@@ -184,6 +191,63 @@ describe( 'Tabs', () => {
 			);
 		} );
 	} );
+	describe( 'Focus Behavior', () => {
+		it( 'should focus on the related TabPanel when pressing the Tab key', async () => {
+			const user = userEvent.setup();
+
+			render( <UncontrolledTabs tabs={ TABS } /> );
+
+			const selectedTabPanel = await screen.findByRole( 'tabpanel' );
+
+			// Tab should initially focus the first tab in the tablist, which
+			// is Alpha.
+			await user.keyboard( '[Tab]' );
+			expect(
+				await screen.findByRole( 'tab', { name: 'Alpha' } )
+			).toHaveFocus();
+
+			// By default the tabpanel should receive focus
+			await user.keyboard( '[Tab]' );
+			expect( selectedTabPanel ).toHaveFocus();
```

Comment:
In your testing instructions, you have the following step: 
>While the tabpanel is focused, inspect the element and confirm the component's new :focus-visible style is overriding the browser defaults.

Since you mentioned it in your testing instructions, maybe it would be worth including in the test? Something like: 

`expect( selectedTabPanel ).toHaveAttribute( 'data-focus-visible' );`

Reply:
The `tabpanel` element receiving `:focus-visible` isn't new in this case, so I don't want to add a test for it. The change around `:focus-visible` for this PR is that when the `tab-panel` gains that attribute, it gets styled by [our new CSS](https://github.com/WordPress/gutenberg/blob/ec47c3743e98ce4599ea8d1803edd16addb4be08/packages/components/src/tabs/styles.ts#L106-L118), rather than the browser defaults, which is what was styling it previously :)

Reply:
Thanks for clarifying! 👍